src/geometry/GeometryEvaluator.cc

```diff
@@ -134,6 +134,11 @@ GeometryEvaluator::ResultObject GeometryEvaluator::applyToChildren3D(const Abstr
   if (children.empty()) return {};
 
   if (op == OpenSCADOperator::HULL) {
+#if ENABLE_MANIFOLD
+    if (RenderSettings::inst()->backend3D == RenderBackend3D::ManifoldBackend) {
+      return ResultObject::mutableResult(std::shared_ptr<Geometry>(ManifoldUtils::applyHullManifold(children)));
+    }
+#endif
```

Comment:
See my other comment about using vertices as Hull() argument. If you end up doing that, it might make sense to fold this implementation into applyHull(), and deal with it there (or refactor that into collection + execution functions).